### PR TITLE
Remove stale bot

### PR DESCRIPTION
**Pull request by XVilka**
**Your checklist for this pull request**
- [ ] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [ ] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

It's annoying and didn't anything actually useful for a year I observed this specimen. I propose removing it completely. See also https://drewdevault.com/2021/10/26/stalebot.html

...

**Test plan**

...

**Closing issues**

...

### CODE REVIEW

**Review by wargio — APPROVED** [review]

**Comment by wargio** [comment]
wait for more approvals before merging

**Review by thestr4ng3r — APPROVED** [review]

**Comment by ret2libc** [comment]
What if we just restrict the issues it applies to? Like, if the issue has enhancement tag it won't apply. Maybe we can even increase the time before marking things stale, but IMHO it is useful to review things that are just there without any activity. It allowed me to find fixed issues or things that didn't make sense anymore or issues that were not well explained (thus no action was ever done).

Anyway, no strong opinions.

**Comment by XVilka** [comment]
We can always enable it back, if necessary.

**Pull request merged**
